src/components/RichTextEditor/RichTextEditor.css

```diff
@@ -33,12 +33,11 @@
 }
 
 .menu-bar-container-hidden {
-  height: 0;
+  height: var(--menu-bar-height-hidden, auto);
   visibility: hidden;
 }
 
 .menu-bar-container-visible {
-  height: auto;
```

Comment:
Could you please add a variable instead of remove this? This way we decrease the chance of breaking something on the projects we have ads installed.

Reply:
@raphaelsattler there is already a variable to control this value, is the `--menu-bar-height`.